ui/app/src/views/core/Core.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from 'react';
+import { HeaderView } from './Header';
+import { FooterView } from './Footer';
+
+export interface CoreProps {
+  children: ReactNode;
+}
+
+export default function CoreView(props: CoreProps) {
+  return (
+    <div>
+      <HeaderView />
```

Comment:
Ah, I assumed the `View` part in there was similar to why you had the `View` in your components, because `Dashboard` is already a widely used term in the rest of the project as well.

Reply:
I think of `View` basically being synonymous with "Page" (which is why I used that naming convention for `DashboardView` since it's a main "page" in the app). Not sure how many views we're going to have initially to be honest (Dashboard may be the only one for awhile). But since these are just components, totally cool with just calling them `Header` and `Footer` if you want?

Reply:
I think we should have these views : 
* home page (no idea what we would have inside, but could be cool)
* the dashboard display (of course!)
* a page to display the datasource
* Perhaps the metric explorer ? (could be cool to have it)

Probably I will call them `Header` and `Footer` since they are just components and not the entire page.